internal/scaffold/cmd_test.go

```diff
@@ -150,7 +151,31 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = serveCRMetrics(cfg); err != nil {
+	// Add the Metrics Service
+	addMetrics(cfg, ctx, namespace)
+
+	log.Info("Starting the Cmd.")
+
+	// Start the Cmd
+	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
+		log.Error(err, "Manager exited non-zero")
+		os.Exit(1)
+	}
+}
+
+// addMetrics will create the Services and Service Monitors to allow the operator export the metrics by using
+// the Prometheus operator
+func addMetrics(cfg *rest.Config, ctx context.Context, namespace string) {
```

Comment:
```suggestion
func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
```
`context.Context` should be the first argument, by convention.